### ISSUE TITLE
Default newton absolute error for dune seems wrong

### ISSUE THREAD

**Issue by SoilRos** [spatial-model-editor/spatial-model-editor#956]
**Is your feature request related to a problem? Please describe.**
The current examples (and expected inputs) have mesh sizes that are very small in absolute values. This, turns into absolute values that are also small for convergence criterions. The dune simulator in the editor has a default that doesn't seems suited for these kind of inputs: 1.00000e-12. Basically all of the examples that I tried[^1] "stay" in a steady state when simulated with the dune simulator due to this option.

**Describe the solution you'd like**
I would like that the dune simulation works "out-of-the-box" and doesn't need to be fine tuned for the common case. The absolute error checking in dune-copasi is optional, meaning whenever this value is not set, there won't be a check against the absolute value. Since most of the simulations here are expected to be dynamic, I would set this to empty as a default and let people change it when they know a sensible value for their system.

[^1]: except of the simple diffusion, probably because this does not use the non-linear solver

**Comment by lkeegan** [comment]
Thanks for looking into this - this sounds like a good solution to me

**Comment by SoilRos** [comment]
Hmm, I was thinking about this again. When the solution converges towards the steady state, the absence of an absolute tolerance will make the problem more and more difficult to resolve until it cannot reach the relative tolerance :thinking:. How do you solve this in the pixel simulator? How about an absolute tolerance that is a percentage of the total amount of species in the whole domain when they are equal to one or another expected value that the user can set up?

**Comment by lkeegan** [comment]
I agree this is a problem - Pixel uses explicit solvers so this issue doesn't come up there.
I guess it's not possible to have a heuristic for a good absolute tolerance for any model in general, but scaling it with species amount sounds like a good idea to me.
Are the units of the absolute tolerance species amount?
The user units in the GUI are typically biology concentration units, e.g. mmol/mL - so 1 in these units can be some crazy number in amount/volume in the dune-copasi model

**Comment by SoilRos** [comment]
The tolerance check should come evaluating the weak formulation integral with the current trial. So if I am not messing up something, it should be something like mmol/s (assuming the transient part has no scaling units, is that correct?). Sometimes one changes the equation to multiply by the dt instead of dividing by it but that must be handled internally by scaling the tolerance too. I need to double check this in our solvers.
